src/ui/components/Scanner/Scanner.tsx

```diff
@@ -126,6 +119,12 @@ const Scanner = forwardRef(
       // @TODO - foconnor: instead of setting the optype to idle we should
       // have a loading screen with "waiting for server..." etc,
       // and it can update to an error if the QR is invalid with a re-scan btn
+      if (currentOperation === OperationType.SCAN_WALLET_CONNECTION) {
+        dispatch(setCurrentOperation(OperationType.IDLE));
```

Comment:
I think we should remove this `dispatch(setCurrentOperation(OperationType.IDLE));` and the other one at line 128 and put it a the bottom of `processValue`.

Reply:
Because I don't want continue call to `Agent.agent.connections.connectByOobiUrl`, I return to break `processValue` here and I think we need set current operation to idle